compiler/src/iree/compiler/InputConversion/MHLO/Passes.cpp

```diff
@@ -35,6 +35,12 @@ static llvm::cl::opt<bool> clDemoteF64ToF32(
     llvm::cl::desc(
         "Converts all MHLO f64 ops and values into f32 counterparts."),
     llvm::cl::init(true));
+static llvm::cl::opt<bool> clPromoteBF16ToF32(
+    "iree-mhlo-promote-bf16-to-f32",
```

Comment:
I think you are following naming convention here, but is there anything MHLO specific?

Reply:
No, just cargo culting.  Should I also move these to a different directory?

Reply:
Nope, that's good, it fits the flow, just checking.